src/main/java/frc/robot/util/Vision.java

```diff
@@ -501,5 +505,9 @@ public void enable(boolean enable){
     public void setOnlyUse(int id){
       onlyUse = id;
     }
+
+    public PhotonCamera getCamera(){
+      return camera;
+    }
```

Comment:
You shouldn't need to access this. It's also in a private class, so adding a public method to get the camera outside of Vision.java won't work. All instance variables are accessible in Vision.java without methods.